webapp/channels/src/i18n/en.json

```diff
@@ -1462,6 +1462,7 @@
   "admin.logs.options": "Options",
   "admin.logs.prev": "Previous",
   "admin.logs.ReloadLogs": "Reload Logs",
+  "admin.logs.DownloadLogs": "Download Logs",
```

Comment:
Another CI check is failing because this string isn't in the right place because this file is sorted alphabetically. If you go to the `webapp` folder and run `npm run i18n-extract -w channels`, it'll regenerate this file and sort it into the right order